solidity/contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -105,7 +105,7 @@ contract LiquidityProtection is ILiquidityProtection, Utils, Owned, ReentrancyGu
         store = ILiquidityProtectionStore(_contractAddresses[1]);
         stats = ILiquidityProtectionStats(_contractAddresses[2]);
         systemStore = ILiquidityProtectionSystemStore(_contractAddresses[3]);
-        wallet = ITokenHolder(_contractAddresses[4]);
+        wallet = ITokenHolder(payable(_contractAddresses[4]));
```

Comment:
What does `payable` achieve here? I cannot see how this would resolve any compilation error or warning (let alone, runtime behavior).

Reply:
It is, since `TokenHandler` is now `payable`

Reply:
```
CompileError: /Users/leonid/work/bancor/contracts/solidity/contracts/liquidity-protection/LiquidityProtection.sol:108:18: TypeError: Explicit type conversion not allowed from non-payable "address" to "contract ITokenHolder", which has a payable fallback function.
        wallet = ITokenHolder(_contractAddresses[4]);
                 ^---------------------------------^
```